crates/dbsp/src/storage/backend/posixio_impl.rs

```diff
@@ -300,16 +300,35 @@ impl StorageBackend for PosixBackend {
     }
 }
 
+pub(crate) struct PosixBackendFactory;
+impl StorageBackendFactory for PosixBackendFactory {
+    fn backend(&self) -> &'static str {
+        "default"
```

Comment:
Can this be "posixio"?

Reply:
POSIX is a bad name for this struct because IIRC it should work on Windows too which isn't posix